search/elastic.py

```diff
@@ -270,7 +270,8 @@ def _clear_mapping(self, doc_type):
 
     def __init__(self, index=None):
         super(ElasticSearchEngine, self).__init__(index)
-        self._es = getattr(settings, "ELASTIC_SEARCH_IMPL", Elasticsearch)()
+        es_config = getattr(settings, "ELASTIC_SEARCH_CONFIG", [{}])
```

Comment:
I'm a little confused as to why this takes in a list of hosts. Is that just for fallbacks?